src/main/java/com/kodilla/ecommercee/domain/GroupDto.java

```diff
@@ -0,0 +1,15 @@
+package com.kodilla.ecommercee.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import java.util.Map;
+
+
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+public class GroupDto {
```

Comment:
ja akurat zgadzam się z Robertem C. Martnem, że klasy typu Dto to jest zwykła struktura - nośnik danych. Dlatego robienie do nich konstruktorów, getterów czy setterów jest po prostu totalnie bez sensu. Tutaj robotę za nas robi Lombok, ale i tak mam do tego mieszane uczucia. Zostaw do jedna, bo jest to zgodne z treścią kursu - przekazuje Ci tylko moje personalne odczucia.

Reply:
ok. Rozumiem.